App/src/pages/DeleteCourse.tsx

```diff
@@ -0,0 +1,186 @@
+import Header from "../shared/Header";
+import {
+    Flex,
+    Center,
+    Text,
+    Heading,
+    FormControl,
+    FormLabel,
+    Stack,
+    NumberInput,
+    NumberInputField,
+    Box,
+    Textarea,
+    useToast,
+    FormErrorMessage,
+} from "@chakra-ui/react";
+import { useState, useEffect } from "react";
+import { deleteCourse, getAllCourseSettings } from "../services/course";
+import { AllCourseSettings, CourseCareer } from "../models/course";
+import AutocompleteInput from "../components/Select";
+import { showToast } from "./../utils/toastUtils";
+import Button from "../components/Button";
+
+export default function DeleteCourse() {
+    const toast = useToast();
+
+    const [isLoading, toggleLoading] = useState(false);
+    const [formSubmitted, setFormSubmitted] = useState(false);
+    const [courseSubjectError, setCourseSubjectError] = useState(true);
+    const [courseCatalogError, setCourseCatalogError] = useState(true);
+    const [courseCareersError, setCourseCareersError] = useState(true);
+    const [rationalError, setRationalError] = useState(true);
+
+    const [allCourseSettings, setAllCourseSettings] = useState<AllCourseSettings>(null);
+    const [subject, setSubject] = useState("");
+    const [catalog, setCatalog] = useState("");
+    const [rationale, setRationale] = useState("");
+    const [courseCareer, setCouresCareer] = useState<CourseCareer>(null);
+    const [courseCareers, setCourseCareers] = useState<string[]>([]);
+
+    const handleChangeCourseCareer = (value: string) => {
```

Comment:
I haven't run it, but code looks good in general. I'm just curious, what's the point of worrying about course careers here at all? It's not part of the API.

Reply:
You are right, I don't think we need course career. I removed it.